compiler/test/runnable/mangle.d

```diff
@@ -1,6 +1,7 @@
 // PERMUTE_ARGS:
 // EXTRA_SOURCES: imports/mangle10077.d
 // EXTRA_FILES: imports/testmangle.d
+// UNICODE_NAMES:
```

Comment:
This test already has a `//UTF-8 chars` comment here: https://github.com/dlang/dmd/pull/16136/files#diff-76a096d23b36bb26ba1e9a687500e3b2733abf078b8d7edfd7b143c72330544eL46-R48